### DISCUSSION TITLE
Proposal: Publish Console Apps as Web Jobs

### DISCUSSION THREAD

**Discussion by csharpfritz** [isaacrlevin/WorkFlowGenerator#20] [Ideas]
If a console application is detected, allow it to be deployed to App Service as a Web Job

**Comment by isaacrlevin** [comment]
Thanks @csharpfritz. Are there any other targets in Azure for console apps?

**Reply by csharpfritz** [reply]
Certainly they could be packaged in a Docker container and shipped to ACA, AKS, and other container services
